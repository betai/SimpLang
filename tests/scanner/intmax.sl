let main x =
  9223372036854775807
end
