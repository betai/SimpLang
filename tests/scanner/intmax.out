keyword let
identifier main
identifier x
operator =
integer 9223372036854775807
keyword end
